### DISCUSSION TITLE
/usr/local/lib/ruby/3.3.0/fileutils.rb:402:in `mkdir': Permission denied

### DISCUSSION THREAD

**Discussion by baozoudashijian** [rails/docked#40] [Q&A]
when i was running "docked rails new weblog" it throw a error it like this "/usr/local/lib/ruby/3.3.0/fileutils.rb:402:in `mkdir': Permission denied @ dir_s_mkdir - /rails/weblog (Errno::EACCES)", 
![image](https://github.com/user-attachments/assets/1c4be24e-f13c-40ab-966e-d71879340ab0)
please help me solove this problem

**Comment by rob-hunt** [comment]
I get the same error.  My first thought was that I needed to add that directory in docker settings/resources/file sharing, but I was actually already in a directory that had been shared in this manner.  Running under Linux Mint 21.3 if that's significant
